### PR TITLE
Add ecs and kube cluster name tags

### PR DESCRIPTION
**Pull request by ahmed-mez**
### What does this PR do?

- Add `ecs_cluster_name` and `kube_cluster_name` in addition to `cluster_name`
- Add a new config `disable_cluster_name_tag_key` to disable sending the `cluster_name` tag

### Motivation

- `cluster_name` can refer to other clusters in some integrations, not the orchestrator cluster
- `ecs_cluster_name` and `kube_cluster_name` are needed for the orchestrator explorer

### CODE REVIEW

File: pkg/metadata/host/host_tags.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func getHostTags() *tags {
 
 	clusterName := clustername.GetClusterName()
 	if len(clusterName) != 0 {
-		hostTags = appendToHostTags(hostTags, []string{"cluster_name:" + clusterName})
+		hostTags = appendToHostTags(hostTags, []string{"cluster_name:" + clusterName, "kube_cluster_name:" + clusterName})
```
Comment (mfpierre) [inline pkg/metadata/host/host_tags.go:72]: In cases `cluster_name` conflicts with other integrations, should we add an option to remove this tag and keep only `kube_cluster_name`?
And in this case should put a warning somewhere advertising to activate this option and using this new tag, WDYT?

Comment (ahmed-mez) [reply pkg/metadata/host/host_tags.go:72]: yes could be useful to have the option to remove the tag, on it

File: releasenotes/notes/orchestrator-tag-9e9a7da025799f3a.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Introduce `kube_cluster_name` and `ecs_cluster_name` tags in addition to `cluster_name`.
```
Comment (mfpierre) [inline releasenotes/notes/orchestrator-tag-9e9a7da025799f3a.yaml:4]: can you also add the new option to the changelog?

Comment (ahmed-mez) [reply releasenotes/notes/orchestrator-tag-9e9a7da025799f3a.yaml:4]: done

**Review by mfpierre — APPROVED** [review]
LGTM 👍

**Review by remeh — APPROVED** [review]

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**